src/components/Toolbelt/Toolbelt.sass

```diff
@@ -1,4 +1,12 @@
 .Toolbelt
+  .visually_hidden:not(:focus):not(:active)
+    clip: rect(0 0 0 0)
+    clip-path: inset(50%)
+    height: 1px
+    overflow: hidden
+    position: absolute
+    white-space: nowrap
+    width: 1px
```

Comment:
this can probably live in more of a "base" stylesheet if there is one, also i didn't make these styles up so here's a bit more about them https://css-tricks.com/inclusively-hidden/

Reply:
I love this! This seems generally useful so let's move it to (what is effectively) the base stylesheet: https://github.com/jeremyckahn/farmhand/blob/develop/src/Farmhand.sass

Reply:
This is done in https://github.com/jeremyckahn/farmhand/commit/fd24e6fdd9d600683fb31c9f92f5df6415a68e46!